docs/examples/distributed/002_multi_gpu/job.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#SBATCH --gpus-per-task=rtx8000:1
+#SBATCH --cpus-per-task=4
+#SBATCH --ntasks-per-node=4
+#SBATCH --mem=16G
+#SBATCH --time=00:15:00
+
+
+# Echo time and hostname into log
+echo "Date:     $(date)"
+echo "Hostname: $(hostname)"
+
+
+# Ensure only anaconda/3 module loaded.
+module --quiet purge
+# This example uses Conda to manage package dependencies.
+# See https://docs.mila.quebec/Userguide.html#conda for more information.
+module load anaconda/3
+module load cuda/11.7
+
+# Creating the environment for the first time:
+# conda create -y -n pytorch python=3.9 pytorch torchvision torchaudio \
+#     pytorch-cuda=11.7 -c pytorch -c nvidia
+# Other conda packages:
+# conda install -y -n pytorch -c conda-forge rich tqdm
+
+# Activate pre-existing environment.
+conda activate pytorch
+
+
+# Stage dataset into $SLURM_TMPDIR
+mkdir -p $SLURM_TMPDIR/data
+ln -s /network/datasets/cifar10/cifar-10-python.tar.gz $SLURM_TMPDIR/data/
```

Comment:
`ln -s` fait un symbolic link. Ceci redirigera tous les IOs sur `/network/datasets`. À ne pas recommender en général....

Reply:
Je le fais juste pour l'archive par contre, donc c'est correct ici?

Reply:
_Okay_, ici et dans ce contexte, oui c'est techniquement la meilleure chose à faire, mais ça mène en erreur parce que ça ne s'applique qu'au contexte des torchvision DataLoaders. Si quelqu'un fait copié-collé de ce template pour son test avec un méga dataset alternatif, et que le data loader sait comment lire "direct" le Zip/Tar whatever sans unpack (comme [ici](https://github.com/MLRichter/ConvNeXt/blob/benchmark/nested_tarbal_parser.py)), les I/Os iront directement sur `$SCRATCH`, sans que l'utilisateur plus novice n'en devienne conscient.

Je préférerais `cp` ici pour symboliser "copie" ici au sens direct, mais peut-être avec un commentaire qui inclut une commande `tar` et `zip`:

```bash
# Stage dataset into $SLURM_TMPDIR
mkdir -p $SLURM_TMPDIR/data
cp /network/datasets/cifar10/cifar-10-python.tar.gz $SLURM_TMPDIR/data/
# General-purpose alternatives combining copy and unpack:
#     unzip   /network/datasets/some/file.zip -d $SLURM_TMPDIR/data/
#     tar -xf /network/datasets/some/file.tar -C $SLURM_TMPDIR/data/
```

Reply:
Fixed in 0e6c98951e10e717b25f3770f0fad0382e3a6797